anycloud/cli/src/deploy/mod.rs

```diff
@@ -213,48 +215,38 @@ fn get_aws_cli_creds() -> Result<AWSCLICredentialsFile, String> {
 
 pub async fn add_cred() -> String {
   let mut credentials = get_creds().await;
-  let clouds = vec!["AWS", "GCP", "Azure"];
-  let selection = Select::with_theme(&ColorfulTheme::default())
-    .with_prompt("Pick cloud provider for the new Credential")
-    .items(&clouds)
-    .default(0)
-    .interact()
-    .unwrap();
-  let cloud = clouds[selection];
-  let cred_name = Input::with_theme(&ColorfulTheme::default())
-    .with_prompt("Name for new Credential")
-    .validate_with(|input: &String| -> Result<(), &str> {
+  let clouds = vec!["AWS".to_string(), "GCP".to_string(), "Azure".to_string()];
```

Comment:
i understand *why* there's a need for both `&str` and `String` but it never fails to disappoint me whenever things look like this because it could be so much easier if there was a way to unify them :(

Reply:
agree